src/IOTWrapper/IOT.ts

```diff
@@ -68,7 +68,7 @@ class IOT {
         if(this.sync && this.isValid) {
             this.sendBeaconSync(beacon);
         } else if (this.isValid){
-            this.sendBeaconAsync(beacon).catch((err) => { Logger.error(err)});
+            this.sendBeaconAsync(beacon).catch((err) => { Logger.error(JSON.stringify(err))});
```

Comment:
we should really come up with a common error handling pattern and just c/p from someone else on the best way to log errors.